test/Microsoft.Health.Fhir.Tests.Integration/Persistence/FhirStorageTestsFixture.cs

```diff
@@ -31,9 +44,19 @@ public FhirStorageTestsFixture(DataStore dataStore)
 
         public IFhirDataStore DataStore => _scopedStore.Value;
 
+        public IFhirOperationDataStore OperationDataStore => _scopedFhirOperationDataStore.Value;
+
+        public IFhirStorageTestHelper TestHelper => _scopedTestHelper.Value;
+
         public void Dispose()
         {
             _scopedStore?.Dispose();
+            _scopedFhirOperationDataStore?.Dispose();
+            _scopedTestHelper?.Dispose();
```

Comment:
Is Dispose() still called when you implement IAsyncLifetime?

Reply:
Yes Dispose will still be called even if IAsyncLifetime interface is implemented.